tests/test_pidfile.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -
+#
+# This file is part of gunicorn released under the MIT license.
+# See the NOTICE for more information.
+
+import errno
+import sys
+
+try:
+    import unittest.mock as mock
+except ImportError:
+    import mock
+
+import gunicorn.pidfile
+
+
+def mock_patch_builtin(name):
```

Comment:
let just use the latest mock version. There is no need to be conservative for tests imho :)